src/subcommands/logs.ts

```diff
@@ -0,0 +1,160 @@
+// Copyright 2021 Deno Land Inc. All rights reserved. MIT license.
+
+import { wait } from "../../deps.ts";
+import { error } from "../error.ts";
+import { API, APIError } from "../utils/api.ts";
+
+const help = `deployctl logs
+Shows the logs of the given project.
+
+To show the latest logs of a project:
+  deployctl logs --project=helloworld
+
+To show the logs of a particular deployment:
+  deployctl logs --project=helloworld --deployment=1234567890ab
+
+To show the logs of the production deployment:
+  deployctl logs --project=helloworld --prod
+
+USAGE:
+    deployctl logs [OPTIONS] [<PROJECT>]
+
+OPTIONS:
+        --deployment=<DEPLOY_ID>  The id of the deployment you want to get the logs (defaults to latest deployment)
```

Comment:
```suggestion
        --deployment=<DEPLOYMENT_ID>  The id of the deployment you want to get the logs (defaults to latest deployment)
```

We use "deployment_id` to refer to the ID of a deployment. It should be better to be consistent.